src/parse.rs

```diff
@@ -678,10 +678,24 @@ fn dlt_payload<T: NomByteOrder>(
     payload_length: u16,
     arg_cnt: u8,
     is_controll_msg: bool,
+    is_network_trace_msg: bool,
```

Comment:
Using Enums instead of Booleans would make the code a bit cleaner here since `is_controll_msg` and `is_network_trace_msg` can't be true at the same time.

Reply:
Solved by passing on the ext-header's MessageType.